civihr_employee_portal/tests/Webform/CustomComponentKeyHelperTest.php

```diff
@@ -39,10 +39,48 @@ public function testGettingFieldID($key, $expectedGroup, $expectedField) {
   public function testRebuildingKey($original, $groupID, $fieldID, $expected) {
     $this->assertEquals(
       $expected,
-      KeyHelper::rebuildKey($groupID, $fieldID, $original)
+      KeyHelper::rebuildCustomFieldKey($groupID, $fieldID, $original)
     );
   }
 
+  /**
+   * @dataProvider fieldsetKeyProvider
+   *
+   * @param string $key
+   * @param string $expected
+   */
+  public function testGettingGroupIdWillReturnExpectedGroup($key, $expected) {
+    $this->assertEquals($expected, KeyHelper::getCustomFieldsetGroupId($key));
+  }
+
+  /**
+   * @dataProvider fieldsetKeyProvider
+   *
+   * @param string $key
+   * @param bool $expected
```

Comment:
Param type should still be string as above. The conversion to boolean happens inside the function.